src/components/Book.js

```diff
@@ -0,0 +1,30 @@
+import './Book.css';
+import { useState } from 'react';
+import BookInfo from './bookInfo/BookInfo';
+
+const Book = () => {
+  const [bookData, setBookData] = useState([]);
+
+  const handleAddBook = (book) => {
+    setBookData([...bookData, book]);
+  };
+  return (
+    <div>
+      <h2>Book Store</h2>
+      <ul>
+        {bookData.map((book) => (
+          <li key={bookData.length}>
+            {book.bookName}
+            {' '}
+            by
+            {' '}
+            {book.authorName}
+          </li>
```

Comment:
- [ ] It would be very nice if you put this into a separate component **SingleBook** and pass props from here to make your components reusable 😊
- [ ] It would be nice if you put a remove button inside the **SingleBook** component because it is also rendering with every book and allows user to delete that single book 👍

Reply:
Thank you for the review.
I have implemented the requested changes